react/package.json

```diff
@@ -31,7 +31,8 @@
     "husky": "^3.0.9",
     "lint-staged": "^9.4.3",
     "prettier": "^2.2.1",
-    "typescript": "3.9.7"
+    "typescript": "3.9.7",
+    "vtex.search": "https://events--beautycounterqa.myvtex.com/_v/private/typings/linked/v1/vtex.search@2.14.2+build1679084378/public/@types/vtex.search"
```

Comment:
Remove this reference to your linked app

Reply:
hi @polishq , I forgot about it. sorted out!

Reply:
@carlosviniciusananias Please run `yarn` again too, there's still a reference in yarn.lock

Reply:
@carlosviniciusananias ☝️